jerry-core/parser/js/js-parser-expr.c

```diff
@@ -113,6 +113,27 @@ parser_check_invalid_assign (parser_context_t *context_p) /**< context */
   }
 } /* parser_check_invalid_assign */
 
+#if ENABLED (JERRY_ES2015)
+/**
+ * Check and throw an error if the "new.target" is invalid as a left-hand side expression.
+ */
+static void
+parser_check_new_target_lhs (parser_context_t *context_p, /**< parser context */
```

Comment:
This name should be in line with `parser_check_invalid_assign` somehow. You can rename any or both of them of course, but they have a similar function so they should have a similar name.